lib/galaxy/model/__init__.py

```diff
@@ -9008,7 +9008,7 @@ class WorkflowInvocationToSubworkflowInvocationAssociation(Base, Dictifiable, Re
     dict_element_visible_keys = ["id", "workflow_step_id", "workflow_invocation_id", "subworkflow_invocation_id"]
 
 
-class WorkflowInvocationMessage(Base, Dictifiable, Serializable):
+class WorkflowInvocationMessage(Base, Dictifiable):
```

Comment:
Can you undo that commit and turn it into a bug ? I'll implement the missing function, that's a bug that needs to be fixed in older releases.

Reply:
Added type:ignore for now